packages/flutter/test/widgets/buttons_test.dart

```diff
@@ -40,4 +41,102 @@ void main() {
 
     semantics.dispose();
   });
+
+
+  Widget _buttonWidget({Key buttonKey, Key materialKey,
+    Color color, Color highlightColor, Color splashColor,
+    double minWidth = 150.0, double height = 60.0, bool useTheme = false}) {
```

Comment:
Found the doc: https://www.dartlang.org/guides/language/language-tour#default-parameter-values